scipy/signal/ltisys.py

```diff
@@ -121,10 +121,10 @@ def tf2ss(num, den):
     if num.shape[-1] > 0:
         D = num[:, 0]
     else:
-        D = array([], float)
+        D = array([0], float)
```

Comment:
You changed this line to fix this one issue. I wonder if this change can cause the function to work differently on some other inputs (in a potentially bad way). 

So can you explain that it won't cause problems or incompatibilities on any other inputs? My understanding is that `D` with one zero should be equivalent to empty `D`. But can you confirm / comment on that?